war/src/main/js/index.js

```diff
@@ -1,9 +1,11 @@
 import {defineCustomElements} from "ionicons/dist/loader";
+import { DataTable } from "simple-datatables";
 
 defineCustomElements(window, {
   resourcesUrl: '/',
 });
 
+new DataTable("#probes-table");
```

Comment:
this means that the `DataTable` is created on all pages, even when there is no div with the correct id.
Would it be possible to only activate it on the `probes/listing.html`?